delta/sdk/src/main/scala/ch/epfl/bluebrain/nexus/delta/sdk/projects/Projects.scala

```diff
@@ -263,7 +261,7 @@ object Projects {
           IO.raiseError(ProjectIsMarkedForDeletion(c.ref))
         case Some(s)                        =>
           // format: off
-          referenceFinder.raiseIfAny(c.ref) >>
+          validateDeletion.apply(c.ref) >>
```

Comment:
Maybe scala is forcing you, but .apply?

Reply:
IntelliJ is not always very good at renaming things